website/screens/components/table/code/TableCodePage.tsx

```diff
@@ -80,6 +102,65 @@ const sections = [
       </DxcTable>
     ),
   },
+  {
+    title: "DxcTable.ActionsCell",
+    content: (
+      <DxcParagraph>
+        A compound component aimed to be used inside the table.
+      </DxcParagraph>
+    ),
+    subSections: [
+      {
+        title: "Props",
+        content: (
+          <DxcTable>
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Type</th>
+                <th>Description</th>
+                <th>Default</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <DxcFlex
+                    direction="column"
+                    gap="0.25rem"
+                    alignItems="baseline"
+                  >
+                    <StatusTag status="Required">Required</StatusTag>actions
+                  </DxcFlex>
+                </td>
+                <td>
+                  <ExtendedTableCode>{actionsType}</ExtendedTableCode>
+                </td>
+                <td>
+                  <p>
+                    This prop represents a list of interactuable elements that
+                    will work as buttons or as a dropdown. Those with an
+                    <TableCode>icon</TableCode> are treated as buttons. If any
+                    element lacks an <TableCode>icon</TableCode> and includes
+                    <TableCode>options</TableCode>, it is interpreted as a
+                    dropdown. Only the first one with options will be displayed
+                    and no more than three elements will be displayed in total.
+                    In the case of the dropdown the click function will pass the
```

Comment:
I would change to: "Only the first action with including options will be displayed and there is a limit of 3 actions." Same for resultset table.